<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xn297decoderClass</class>
 <widget class="QMainWindow" name="xn297decoderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>503</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>xn297decoder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_PacketSettings">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Packet settings</string>
    </property>
    <widget class="QSpinBox" name="spinBox_addressLength">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_payloadLength">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Address length</string>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Payload length</string>
     </property>
     <property name="text">
      <string>Payload</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_enhanced">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enhanced packet mode</string>
     </property>
     <property name="text">
      <string>Enhanced</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_autoLength">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Packet length set in PCF</string>
     </property>
     <property name="text">
      <string>Auto length</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_bitrate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Bitrate</string>
    </property>
    <widget class="QRadioButton" name="radioButton_bitrate1M">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1 Mbps</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_bitrate250k">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>250 kbps</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_freqControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Frequency control</string>
    </property>
    <widget class="QSpinBox" name="spinBox_fineTune">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>42</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>-300</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>23</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fine tune (kHz)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>23</y>
       <width>39</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_channel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>42</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_frequency">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2400.00 MHz</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_locateGnuradio">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set gnuradio location</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>751</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="maximumBlockCount">
     <number>25000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startStopFlowgraph">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>start gnuradio flow graph</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xn297decoder.qrc"/>
 </resources>
 <connections/>
</ui>
